vendor/engines/saml_authentication/lib/saml_authentication/devise_configurator.rb

```diff
@@ -19,6 +19,9 @@ def configure!
         config.saml_configure do |settings|
           settings.assertion_consumer_service_url = Rails.application.routes.url_helpers.auth_saml_user_session_url
           settings.issuer = Rails.application.routes.url_helpers.metadata_saml_user_session_url
+          Settings.saml.driver&.each do |key, value|
```

Comment:
I can't quite decide if this should go before or after the call to`configure_security`.  On one hand, it would add the flexibility of overriding the security settings in the YAML, but on the other hand, I'm not sure I want those settings to be overridable. I think we can leave it as is for now and cross that bridge if we need to.

I might also consider `Hash(Settings.saml.driver).each`. It'll still handle a `nil` case, and it's a little more explicit about what it expects.

```ruby
Hash(nil) => {}
Hash({}) => {}
Hash({ a: :b }) => { a: :b }
```

Reply:
> I might also consider Hash(Settings.saml.driver).each. It'll still handle a nil case, and it's a little more explicit about what it expects.

That's a good idea; I like that too. Just pushed the amended commit.